lib/primer/view_components/engine.rb

```diff
@@ -12,6 +12,10 @@ class Engine < ::Rails::Engine
         #{root}/app/lib
       ]
 
+      config.primer_view_components = ActiveSupport::OrderedOptions.new
+      config.primer_view_components.force_functional_colors = true
```

Comment:
Does this mean we force functional colors by default?

Reply:
yes, I think we should always follow Primer's latest implementation, which is functional color in this case